core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -2471,6 +2520,17 @@ class SparkContext(config: SparkConf) extends Logging {
     dagScheduler.cancelJobGroup(groupId)
   }
 
+  /**
+   * Cancel active jobs that have the specified tag. See `org.apache.spark.SparkContext.addJobTag`.
```

Comment:
nit: `@see [addJobTag](#addJobTag)`

Reply:
There are no instances of `@see` and `[link](#anchor)` in this file, so I'd rather stick to how it's in the rest of the file.